storage/connect/frcas.h

```diff
@@ -1,117 +1,119 @@
-    case MSG_ACCESS_VIOLATN:  p = "Violation acc�s m�moire";                                                     break;
+/* The non-ASCII literal strings in this file are encoded in the iso-8859-1 / latin-1 character set
+ * (https://en.wikipedia.org/wiki/ISO/IEC_8859-1) */
+    case MSG_ACCESS_VIOLATN:  p = "Violation acc\xe8s m\xe9moire";                                                     break;
     case MSG_ADD_BAD_TYPE:    p = "Ajout d'une valeur de type %s non conforme dans un tableau %s";               break;
```

Comment:
What if we change these characters to utf-8? Is there a test case that fails? That would make it more consistent with `sql/share/errmsg-utf8.txt`

Reply:
Same comment for all storage/connect files.

Reply:
Without knowing the environment in which these strings are supposed to be _used_, it's very hard to say whether this would work.

Presumably, these strings were encoded in `iso-8859-1` because they were intended to be used as log/console messages in the `fr_FR.iso-8859-1` locale, **not** in the `fr_FR.utf-8` locale. But there's no clear documentation on this point that I can find. :disappointed: